sqld/src/http/mod.rs

```diff
@@ -243,7 +258,8 @@ pub async fn run_http<D: Database>(
         tracing::debug!("got request: {} {}", req.method(), req.uri());
     }
 
-    let app = Router::new()
+    let app = Router::new();
+    let app = app
```

Comment:
why this change?

Reply:
no reason, I was struggling with axum type error, and forgot to change it back :)